code/lib/preview-api/src/modules/preview-web/render/StoryRender.ts

```diff
@@ -226,10 +209,22 @@ export class StoryRender<TRenderer extends Renderer> implements Render<TRenderer
           return this.callbacks.showException(error);
         },
         forceRemount: forceRemount || this.notYetRendered,
-        storyContext: renderStoryContext,
-        storyFn: () => unboundStoryFn(renderStoryContext),
+        storyContext: context,
+        storyFn: () => unboundStoryFn(context),
         unboundStoryFn,
       };
+      await this.runPhase(abortSignal, 'loading', async () => {
+        context.loaded = await applyLoaders(context);
```

Comment:
Isn't this a breaking change? Before `loadedContext`, which was the result of `await applyLoaders(..)` was directly destructured into `renderStoryContext` which then was passed to `applyBeforeEach`. Now we assign the result of `applyLoaders` to the `loaded` property.

Before:
```ts
// simplified
let loadedContext = await applyLoaders(...);
let renderStoryContext = { ...loadedContext }
await applyBeforeEach(renderStoryContext);
```

After:
```ts
// simplified
const context = {...}
context.loaded = await applyLoaders(context);
applyBeforeEach(context)
```